lms/templates/discovery/facet.underscore

```diff
@@ -0,0 +1,16 @@
+<h3 class="header-facet">
+    <%= displayName %>
+</h3>
+<ul data-facet="<%= name %>" class="facet-list collapse">
+    <%= options %>
+</ul>
+<% if (listIsHuge) { %>
+    <div class="toggle ">
+        <button class="show-more discovery-button">
+            <%= gettext("MORE...") %>
```

Comment:
@catong we aren't supposed to use "...", are we? I'm also wondering about the all caps.

@dsego Is there a documentation person who has reviewed the text?